deploy/kubevirt-hostpath-provisioner.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubevirt-hostpath-provisioner
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: kubevirt-hostpath-provisioner
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubevirt-hostpath-provisioner
+subjects:
+- kind: ServiceAccount
+  name: kubevirt-hostpath-provisioner-admin
+  namespace: kubevirt-hostpath-provisioner
+roleRef:
+  kind: ClusterRole
+  name: kubevirt-hostpath-provisioner
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: kubevirt-hostpath-provisioner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "create", "delete"]
+
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch", "update"]
+
+  - apiGroups: ["storage.k8s.io"]
+    resources: ["storageclasses"]
+    verbs: ["get", "list", "watch"]
+
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+
+  - apiGroups: [""]
+    resources: ["endpoints"]
+    verbs: ["get", "list", "watch", "create", "update", "patch"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubevirt-hostpath-provisioner-admin
+  namespace: kubevirt-hostpath-provisioner
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kubevirt-hostpath-provisioner
+  labels:
+    k8s-app: kubevirt-hostpath-provisioner
+  namespace: kubevirt-hostpath-provisioner
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kubevirt-hostpath-provisioner
+  template:
+    metadata:
+      labels:
+        k8s-app: kubevirt-hostpath-provisioner
+    spec:
+      serviceAccountName: kubevirt-hostpath-provisioner-admin
+      containers:
+        - name: kubevirt-hostpath-provisioner
+          image: jgriffith/hpp
```

Comment:
This is just my dev image, we'll update this later with the offiical image from kubevirt